types/index.d.ts

```diff
@@ -50,7 +50,9 @@ declare class HCaptcha extends React.Component<HCaptchaProps, HCaptchaState> {
   getResponse(): string;
   setData(data: object): void;
   isReady(): boolean;
-  execute(opts?: { async?: boolean, rqdata?: string }): Promise<ExecuteResponse> | void;
+  execute(opts: { async: true, rqdata?: string }): Promise<ExecuteResponse>;
+  execute(opts?: { async: false, rqdata?: string }): void;
+  execute(opts: { async: boolean, rqdata?: string }): Promise<ExecuteResponse> | void;
```

Comment:
let's leave `opts?:` and `async?:` in the last one, @JAD3N 

```
execute(opts?: { async?: boolean, rqdata?: string }): Promise<ExecuteResponse> | void;
```

Reply:
Wouldn't that be incorrect typing wise? I figure not specifying opts should always be returning a void type unless I'm missing some context.

Reply:
we actually have the void specified
also need to leave `async?: boolean`

Reply:
Ok, np